src/common/transformations/src/transformations/common_optimizations/pull_through_reduce.cpp

```diff
@@ -111,6 +111,9 @@ ov::pass::PullUnsqueezeThroughReduce::PullUnsqueezeThroughReduce() {
     matcher_pass_callback callback = [=](pattern::Matcher& m) {
         auto& pattern_map = m.get_pattern_value_map();
         const auto input_node = pattern_map.at(input);
+        if (input_node.get_node()->get_output_size() > 1) {
```

Comment:
why do we need this check here?
input_node is an Output<Node>, the transformation should work

Reply:
that's right, thank you!